components/onboarding_tasklist/onboarding_tasklist_completed.tsx

```diff
@@ -119,8 +122,20 @@ const Completed = (props: Props): JSX.Element => {
     const isPrevLicensed = prevTrialLicense?.IsLicensed;
     const isCurrentLicensed = license?.IsLicensed;
 
+    // Cloud conditions
+    const subscription = useSelector((state: GlobalState) => state.entities.cloud.subscription);
+    const isCloud = license?.Cloud === 'true';
+    const isFreeTrial = subscription?.is_free_trial === 'true';
+    const hadPrevCloudTrial = subscription?.is_free_trial === 'false' && subscription?.trial_end_at > 0;
+    const isCloudFreeEnabled = useSelector(cloudFreeEnabled);
+    const isCloudFreePaidSubscription = isCloud && isCloudFreeEnabled && license?.SkuShortName !== LicenseSkus.Starter && !isFreeTrial;
```

Comment:
nit: it's confusing Cloud Free and Paid in the name - is it free or paid? 🤣 

```suggestion
    const isCloudPaidSubscription = isCloud && isCloudFreeEnabled && license?.SkuShortName !== LicenseSkus.Starter && !isFreeTrial;
```